### PR TITLE
Second commit in local branh

### PR DESCRIPTION
**Pull request by SruthiK123**
Testing the pull request

### CODE REVIEW

**Review by bharatrbk — APPROVED** [review]
see again

**Review by ShakilaNatarajan — APPROVED** [review]
Nice Happy code

File: StringExample.java [hunk 1]
```diff
@@ -13,7 +13,7 @@
 		System.out.println("s4: " + s4);
 		System.out.println("s5: " + s5);
 		
-		//showing effect of precedence
```
Comment (SruthiK123) [inline StringExample.java:16]: Modified code

**Review by SruthiK123 — COMMENTED** [review]
> see again

ok

**Pull request merged**